client/scss/components/_grid.legacy.scss

```diff
@@ -109,7 +109,7 @@
     @include column(12);
   }
 
-  .row {
+  .w-header__main {
```

Comment:
We do not want to change the grid legacy styles - these classes should be kept as is.

Instead we should adjust the styles in `_header.scss` to preserve whatever behaviour was in these mixins (if needed).

As per the linked issues on the main issue - the plan is to deprecate this file completely in the future.